tools/sof-kernel-log-check.sh

```diff
@@ -50,6 +50,12 @@ ignore_str="$ignore_str"'|thermal thermal_zone.*: failed to read out thermal zon
 ignore_str="$ignore_str"'|iwlwifi 0000:00:14\.3: Microcode SW error detected\. Restarting 0x0\.'
 ignore_str="$ignore_str"'|wlo1: authentication with f4:f5:e8:6b:45:bb timed out'
 
+# Test cases on some platforms fail because the false error message:
+#    sof-audio-pci 0000:00:1f.3: status = 0x00000000 panic = 0x00000000
```

Comment:
@aiChaoSONG Please add a reference to https://github.com/thesofproject/sof/issues/3395 in the comment for this filter.

Reply:
@kv2019i Audio controller on different platforms, they have dirrerent PCI ID.
BYT : `00:1b.0 Audio device: Intel Corporation Atom Processor Z36xxx/Z37xxx Series High Definition Audio Controller (rev 11)
`
APL: `00:0e.0 Multimedia audio controller: Intel Corporation Celeron N3350/Pentium N4200/Atom E3900 Series Audio Cluster (rev 0b)`
BDW: `00:03.0 Audio device: Intel Corporation Broadwell-U Audio Controller (rev 09)`
CML/TGL...: `00:1f.3 Multimedia audio controller: Intel Corporation Device a0c8 (rev 20)`

Reply:
@kv2019i Bug reference added.